paymentsheet/src/test/java/com/stripe/android/utils/screenshots/PermutationsFactory.kt

```diff
@@ -0,0 +1,13 @@
+package com.stripe.android.utils.screenshots
+
+object PermutationsFactory {
```

Comment:
I noticed we have a function in PaparazziRule.kt that likely does the same thing. Can we remove this class?

Reply:
Right, I moved it into `PaparazziRule.kt` and didn’t remove it here.